KishanJ-Patel/One-Stop-Shopper
# Render calculated subtotal on cart page

### Context

- This pull request addresses issue OSS-38 where users request the ability to see the subtotal price of the products on the cart page.
- Showing subtotal is a critical feature for the basic shopping experience.

### Changes

- Added subtotal price in the subtotal section on the cart page.

### Testing

- To test this feature:
  1. Open the home page of our web app.
  2. Add a product to the cart by clicking the add-to-cart button on any product card.
  3. Open the cart page by clicking the cart button on the navbar.
  4. Observe the subtotal in the subtotal section.
  5. Add/remove products in the cart to recalculate the subtotal.

### Related Issues:

- Closes `OSS-38`

### Reviewer Requests

- @KishanJ-Patel : For frontend review
- @KishanJ-Patel : For testing and QA
